Formula/racer.rb

```diff
@@ -0,0 +1,33 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on "rust" => :build
+
+  resource "rust_source" do
+    url Formula["rust"].stable.url
+    sha256 Formula["rust"].stable.checksum.hexdigest
+  end
+
+  def install
+    system "cargo", "build", "--release"
+    (libexec/"bin").install "./target/release/racer"
+    (bin/"racer").write_env_script(libexec/"bin/racer", :RUST_SRC_PATH => "#{HOMEBREW_PREFIX}/share/rust_src/current")
+    resource("rust_source").stage do
+      rm_rf ["src/llvm", "src/test", "src/librustdoc", "src/etc/snapshot.pyc"]
+      (share/"rust_src/#{Formula["rust"].stable.version}").install Dir["src/*"]
+    end
+  end
+
+  def post_install
+    ln_sf (share/"rust_src/#{Formula["rust"].stable.version}"), (share/"rust_src/current")
```

Comment:
This doesn't work. it makes a dir named current containing a symlink to 1.9.0 not a symlink named current pointing to 1.9.0.


```
ls -l /usr/local/Cellar/racer/1.1.0/share/rust_src/current
total 8
lrwxr-xr-x  1 camdennarzt  admin     8B 18 Jun 15:09 1.9.0 -> ../1.9.0
```